packages/newspringchurchapp/src/tabs/prayer/index.js

```diff
@@ -1,15 +1,23 @@
+import React from 'react';
 import { createStackNavigator } from 'react-navigation';
 import { tabBarIconNewSpring } from '../tabBarIcon';
+import { AddPrayerFormConnected } from '../../prayer/AddPrayer/AddPrayerForm';
 import Prayer from './Prayer';
 
-export const PrayerNavigator = createStackNavigator(
+const PrayerNavigator = createStackNavigator(
   {
     Prayer,
+    AddPrayerFormConnected: {
+      // TODO: remove the lint disable once react-navigation can work well with memo
+      screen: () => <AddPrayerFormConnected />, // eslint-disable-line react/display-name
```

Comment:
Vote to remove `memo` on this component?

Reply:
oh yea meant to do that after our debate